src/main/resources/assets/nguhcraft/lang/de_de.json

```diff
@@ -0,0 +1,281 @@
+{
+  "block.nguhcraft.acacia_slab_vertical": "Vertikale Akazienholzstufe",
```

Comment:
This file should be added in a separate pr since it’s mostly unrelated to everything else